### PR TITLE
Send useragent

### PR DESCRIPTION
**Pull request by mikaelforeseeti**

### CODE REVIEW

File: securicad/vanguard/client.py [hunk 1]
```diff
@@ -34,7 +35,10 @@ def __init__(self, username, password, url, region="eu-central-1"):
         self.base_url = url
         self.backend_url = f"{self.base_url}/backend"
         self.token = self.authenticate(username, password, region)
-        self.headers = {"Authorization": self.token}
+        self.headers = {
+            "User-Agent" : "Vanguard SDK {}".format(securicad.vanguard.__version__),
```
Comment (maxwalls) [inline securicad/vanguard/client.py:39]: f-string

File: securicad/vanguard/__init__.py [hunk 2]
```diff
@@ -12,12 +12,12 @@
...
-from securicad.vanguard.client import Client
-from enum import Enum
-
-__author__ = "Foreseeti AB"
 __version__ = "0.0.1"
+__author__ = "Foreseeti AB"
+
+from enum import Enum
 
+from securicad.vanguard.client import Client
```
Comment (maxwalls) [inline securicad/vanguard/__init__.py:20]: This change was not necessary?

Comment (mikaelforeseeti) [reply securicad/vanguard/__init__.py:20]: I thought it still was but I tried it and it wasn't.

File: securicad/vanguard/client.py (hunk 1 above)
Comment (mikaelforeseeti) [reply securicad/vanguard/client.py:39]: I got the idea that f-string wouldn't work...but it works fine. Fixed.

File: securicad/vanguard/client.py [hunk 3]
```diff
@@ -34,7 +35,11 @@ def __init__(self, username, password, url, region="eu-central-1"):
         self.base_url = url
         self.backend_url = f"{self.base_url}/backend"
         self.token = self.authenticate(username, password, region)
-        self.headers = {"Authorization": self.token}
+        self.headers = {
+            "User-Agent" : f"Vanguard SDK {securicad.vanguard.__version__}",
+            "Authorization": self.token
+        }
+        print(self.headers)
```
Comment (maxwalls) [inline securicad/vanguard/client.py:42]: Remove print

**Pull request merged**